curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md

```diff
@@ -14,11 +14,15 @@ If you import styles from a stylesheet, it isn't much different at all. You appl
 
 You apply inline styles to JSX elements similar to how you do it in HTML, but with a few JSX differences. Here's an example of an inline style in HTML:
 
-`<div style="color: yellow; font-size: 16px">Mellow Yellow</div>`
+```html
+<div style="color: yellow; font-size: 16px">Mellow Yellow</div>
+```
 
 JSX elements use the `style` attribute, but because of the way JSX is transpiled, you can't set the value to a `string`. Instead, you set it equal to a JavaScript `object`. Here's an example:
 
-`<div style={{color: "yellow", fontSize: 16}}>Mellow Yellow</div>`
+```html
```

Comment:
````suggestion
```jsx
````

Reply:
@LaurentLabine You have not implemented this requested change, so I am not sure why you marked it Resolved.